pkg/schedule/filter/counter.go

```diff
@@ -83,8 +83,8 @@ var scopes = [scopeLen]string{
 
 	"evict-leader-scheduler",
 	"region-scatter",
-	"replica-checker",
-	"rule-checker",
+	"replica_checker",
```

Comment:
Can we unify the `-` and `_`?

Reply:
scheduler will be unified in the next pr~

Reply:
The monitor might also need to be changed.

Reply:
Do you mean the checker's monitor?

Reply:
I'm not sure about that, maybe scheduler or maybe checker

Reply:
The scheduler name has not been changed.

The checker monitor is no need to change, after I checked. They are all in the form of xxx_checker.

Reply:
Just a reminder